src/BackupDestination/Backup.php

```diff
@@ -60,7 +60,7 @@ public function date(): Carbon
     /**
      * Get the size in bytes.
      */
-    public function size(): int
+    public function size(): flaot
```

Comment:
This type is wrong. Could you also add a test that hits this code, so we can't make a mistake in the future?

Reply:
I'm on it. I will rewrite the unit tests so they will pass with the given float instead of integer.

Reply:
New commit added where the typo is fixed and two tests where added for Backup and BackupCollection.